python/turbine_models/custom_models/sd_inference/unet.py

```diff
@@ -100,7 +182,7 @@ def export_unet_model(
     if hf_model_name == "stabilityai/stable-diffusion-2-1-base":
         encoder_hidden_states_sizes = (2, 77, 1024)
 
-    sample = (batch_size, unet_model.unet.config.in_channels, height // 8, width // 8)
```

Comment:
Do we not use the scaling as such for controlled unet? Can this be made conditional?